packages/cli/src/service/templates/client.tpl.ts

```diff
@@ -2,25 +2,23 @@ import { generator, inject, toFile, when, after } from '@feathershq/pinion'
 import { ServiceGeneratorContext } from '../index'
 
 const schemaImports = ({ upperName, folder, fileName }: ServiceGeneratorContext) => /* ts */ `import type {
+  ${upperName},
   ${upperName}Data,
-  ${upperName}Patch,
-  ${upperName}Result,
   ${upperName}Query,
-} from './services/${folder.join('/')}/${fileName}.schema'
+} from './services/${folder.join('/')}/${fileName}.service'
 
```

Comment:
If we get rid of this extra line break it will keep the import and export lines grouped by service.  The current whitespace visually mixes the services up due to the whitespace between the import and export.